### DISCUSSION TITLE
Any harm in changing task from "At logon" to "At startup"?

### DISCUSSION THREAD

**Discussion by crawfish3141** [Rem0o/FanControl.Releases#911] [Q&A]
One of my machines is a headless server, and normally the only logon will be via Remote Desktop, which may not happen for a long time. Fan Control's task is configured to run at "Logon of any user", and I'm wondering if I can change it to "At startup", so it'll run when I turn the server on, without ever logging on.

**Comment by Rem0o** [comment]
You can try it, but that's unexplored territory on my end. The program needs user escalation to run, so that might cause issue.
